Fix spec for string gaps in multiline strings (#686)
ghc-proposals/ghc-proposals: proposals/0569-multiline-strings.rst

  -- equivalent to
  s' = "a b c d e\nf g"

> But a string gap starting at the beginning of a line is included in the whitespace calculation.

Reviewer: ```suggestion
But a string gap starting at the beginning of a line counts as non-whitespace in the whitespace calculation.
```
